Created team member component

dalpao opened the pull request:
- Also added .DS_Store to the .gitignore at the root of the project

dalpao committed:
add DS_Store to .gitignore

dalpao committed:
create team member component

soojinhwang reviewed (commented):
Hi Paola, did you pull from the staging branch before you started creating these?
Because Phoebe already created a "utils" folder inside the "src" folder yesterday where we should be putting all the components in, so I think adding another folder called "components" is going to be confusing in the long-run

soojinhwang reviewed (changes requested):
I think also it would be good to separate out the TeamMember component into a separate smaller component, and then create another component/page for About Us and use props.
 
 So something more like this (according to ChatGPT) for the team member component:
 
![image](https://github.com/user-attachments/assets/c9c2475c-9a3d-4bc6-bc29-8e80d4058e46)

dalpao committed:
rename components, remove package-lock.json, move utils folder

dalpao commented:
I have renamed the components with capital letters, removed the package-lock.json at the top level of the repo, and moved the utils folder inside the src folder. I will refactor the Team.js component once I create the About.js component.
